### DISCUSSION TITLE
Can't Convert EWZ4 to EPUB

### DISCUSSION THREAD

**Discussion by JGatesway** [noDRM/DeDRM_tools#620] [General]
Not sure what I'm doing wrong, but I have the latest version of Calibre and the latest DeDRM tool installed. The book is not a wrapped PDF.

Any ideas?

**Comment by ElleKayEm** [comment]
You mean an AZW4 which is a pdf?  Try extracting the pdf with the Kindle Unpack plugin.

**Comment by JGatesway** [comment]
Exactly!  I can't figure out how to employ (deploy?) the KindleUnpack plugin in Calibre!  Do I put PDF as the output file or EPUB?

I'm so sorry to ask such a basic question!

**Reply by ElleKayEm** [reply]
Select the book.  Press the KindleUnpack button on the main menu and there should be an option for extract pdf.

**Comment by JGatesway** [comment]
Yes, but the "Unpack AWZ4" button is greyed out.  There is a lock by the drop down menu and it has an arrow that points to the greyed out "Unpack AWZ4".   

Yikes.  Thanks for your help!

**Reply by ElleKayEm** [reply]
A red closed lock?  Then it still has DRM.  How did you download the book?

**Comment by JGatesway** [comment]
Downloaded from Amazon and imported it into Calibre by using the "Add Books" tab.

**Reply by ElleKayEm** [reply]
"Download & transfer" for a Fire?  Books for Fire tablets can't be decrypted.  Only books for E-ink Kindles can be decrypted via the "download & transfer" method (entering their serial number into DeDRM).  I don't know if AZW4 books are available for E-ink Kindles.  I think you'd need to download with Kindle for PC.

**Comment by JGatesway** [comment]
It is not a Fire. I purchased the Kindle book on Amazon and have opened it in my Kindle app for Mac & iOS.  I want to convert it to EPUB and read it on Apple Books.  I can't seem to get rid of the DRM. How do I make sure it is enabled on the Calibre?  I converted all 290 of my books last week and had no issues.  However, I just purchased this book today and can't get it converted.

**Reply by ElleKayEm** [reply]
This post has info on getting books you can decrypt from Kindle for Mac: https://www.mobileread.com/forums/showthread.php?t=361503
